gradle.properties

```diff
@@ -18,17 +18,23 @@
 group=org.apache.spark.k8s.operator
 version=0.1.0
 
-fabric8Version=6.12.1
+# Caution: fabric8 version should be aligned with Spark dependency
+fabric8Version=6.7.2
```

Comment:
This is caused by the co-exist dependency of Spark 3.5.1 .  fabric8 `6.7.2` and `6.12.1` are not binary compatible, it complains even if we exclude that from Spark dependency. 

A reminder has been added here to align the fabric8 version used by Spark and operator. I'll update this when 4.0.0 rcs become available

Reply:
The incompatible was captured via unit tests.